pkg/dataplane/inventory.go

```diff
@@ -349,7 +349,7 @@ func resolveGroupAnsibleVars(template *dataplanev1.NodeTemplate, group *ansible.
 		group.Vars["edpm_telemetry_podman_exporter_image"] = containerImages.EdpmPodmanExporterImage
 	}
 	if template.Ansible.AnsibleVars["edpm_telemetry_openstack_network_exporter_image"] == nil {
-		group.Vars["edpm_telemetry_openstack_network_exporter_image"] = containerImages.EdpmOpenstackNetworkExporterImage
+		group.Vars["edpm_telemetry_openstack_network_exporter_image"] = containerImages.OpenstackNetworkExporterImage
```

Comment:
depending on the order of reconciliation of the CRs when the new operator version starts, like in getOpenstackNetworkExporterImg, we should probably check if `containerImages.OpenstackNetworkExporterImage` not being nil. In the case where the osversion `status.containerImages` was not yet updated to have the OpenstackNetworkExporterImage it might be.